nf_core/pipelines/download.py

```diff
@@ -1843,6 +1888,9 @@ def __init__(
             elif re.search(r"manifest\sunknown", line):
                 self.error_type = self.InvalidTagError(self)
                 break
+            elif re.search(r"ORAS\sSIF\simage\sshould\shave\sa\ssingle\slayer", line):
```

Comment:
But `\s` matches any whitespace character (equivalent to `[\r\n\t\f\v ]`). 

Since I am applying the regular expression to a REST API response forwarded via a wrapper around a terminal emulator, I think it is better to be more inclusive? I just wanted to reduce the risk of encoding problems e.g. on WSL.

Reply:
okay, makes sense.